django/db/models/fields/related.py

```diff
@@ -98,6 +98,13 @@ def do_pending_lookups(sender, **kwargs):
 
 
 class RelatedField(Field):
+    has_relation = True
```

Comment:
This is redundant now; related fields will get this automatically as a result of having a "rel" attribute.